### ISSUE TITLE
cross_entropy

### ISSUE THREAD

**Issue by sousoul** [sousoul/Blog#4]
### 2. 交叉熵的计算有两种形式，以下是两种形式的比较：
- `tf.losses.categorical_crossentropy([1,0],[0.9,0.1])`
此时默认`from_logits=False`，是把 `[0.9,0.1]`当成 `probability` ，直接在 `tf.losses.categorical_crossentropy()`内计算`label`和`probability`的交叉熵；但这样有时会出现数值不稳定（计算出的交叉熵为nan）的情况
#### TensorFlow2.0中将softmax和crossentropy整合在一起以避免数值不稳定的情况，故可把softmax的计算放到crossentropy中，而不用手动调用softmax
- `tf.losses.categorical_crossentropy(labels,[0.9,0.1],from_logits=True)`
加上`from_logits=True`后，把`[0.9,0.1]`当成`logits`,在 `tf.losses.categorical_crossentropy()`内先计算`logits`经过 `softmax` 后的 `probability`，再计算`label`和`probability`的交叉熵；这样不会出现数值不稳定的情况；
